trunk/SUAVE/Methods/Utilities/latin_hypercube_sampling.py

```diff
@@ -0,0 +1,94 @@
+# latin_hypercube_sampling.py
+#
+# Created:  Jul 2016, R. Fenrich (outside of SUAVE code)
+# Modified: Mar 2017, T. MacDonald
+
+import numpy as np
+
+## If needed for mapping to normal distribution:
+#from scipy.stats.distributions import norm 
+
+def latin_hypercube_sampling(num_dimensions,num_samples,criterion=''):
+    
+    n = num_dimensions
+    samples = num_samples
+    
+    segsize = 1./samples
+    lhd = np.zeros((samples,n))
+    
+    if( criterion == "" ): # sample is randomly chosen from within segment
```

Comment:
That's good. I think I have an idea on how to get rid of that loop. I'm not as concerned about this one though.